manifests/mod.pp

```diff
@@ -33,14 +33,14 @@
 #   Specifies a path to the module. Do not manually set this parameter without a special reason.
 #
 define apache::mod (
-  Optional[String] $package       = undef,
-  String $package_ensure          = 'present',
-  Optional[String] $lib           = undef,
-  String $lib_path                = $apache::lib_path,
-  Optional[String] $id            = undef,
-  Optional[String] $path          = undef,
-  Optional[String] $loadfile_name = undef,
-  Optional[Array] $loadfiles      = undef,
+  Optional[String] $package            = undef,
```

Comment:
This file only contains whitespace changes? It would be helpful if that would be split off in its own commit.

Reply:
reverted (next Push)